tools/cldr-apps/src/main/java/org/unicode/cldr/web/DataPage.java

```diff
@@ -560,7 +560,18 @@ public DataRow(String xpath) {
             baselineStatus = resolver.getBaselineStatus();
 
             rawEnglish = comparisonValueFile.getStringValue(xpath);
-            displayName = getBaselineProcessor().processForDisplay(xpath, rawEnglish);
+
+            Output<String> pathWhereFound = new Output<String>(),
```

Comment:
If you are not replacing the getStringValue, then it is slightly faster/cleaner to replace

```
Output<String> pathWhereFound = new Output<String>(),
     localeWhereFound = new Output<String>();
comparisonValueFile.getStringValueWithBailey(xpath, pathWhereFound, localeWhereFound);
final boolean samePath = xpath.equals(pathWhereFound.value);
```
with
```
final CLDRFile.Status status = new Status();
ENGLISH.getSourceLocaleID(path, status);
 final boolean samePath2 = path.equals(status.pathWhereFound);
```
But approving in the interests of getting this done fast.

Reply:
That makes more sense, I didn't understand why we needed the Bailey value at all